askama_shared/src/parser.rs

```diff
@@ -129,11 +129,27 @@ pub(crate) enum Target<'a> {
     Path(Vec<&'a str>),
 }
 
-/// First field is "minus sign was used on the left part of the item".
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub(crate) enum Sign {
```

Comment:
Hmm, I'd prefer to give this more semantic meaning, like:

```rust
enum Whitespace { // Not sure about this name...
    Preserve, // Alternatively, `Keep`, ...
    Trim, // Alternatively, `Strip`, `Skip`, ....
}
```

Reply:
Having thought about this some more, I believe this feature is called "whitespace suppression" in Jinja and I think that's maybe clearer as to what we're doing than `trim`, so maybe the variants should be called `Suppress` and `Preserve`? Maybe the config option should be `whitespace_handling = "Preserve"`, too?

Reply:
Sounds like a good idea. For the config, I think a boolean would be better, but renaming is ok I guess (I took the name from `flask`). What do you think about `preserve_whitespace`?

Reply:
I feel like `suppress_whitespace` (flipping it) would be better because not suppressing whitespace is (a) the default and (b) more natural. What do you think?

Reply:
Sounds good to me!